src/partials/layout/_footer.html

```diff
@@ -0,0 +1,14 @@
+<footer class="footer">
+  <div class="container">
+    <div class="footer-wrapper">
+      <p class="footer-wrapper__text">© 2020 | All Rights Reserved |</p>
+      <p class="footer-wrapper__text">
+        Developed with<svg class="footer-wrapper__icon" width="14" height="13">
+          <use href="./images/sprite.svg#icon-heart"></use>
+        </svg>
+        <span>by</span>
+        <span class="footer-wrapper__about-developers">GoIT Students</span>
```

Comment:
Add mix classes "link footer-wrapper__link" here and add necessary styling to footer-wrapper__link, remove footer-wrapper__about-developers class.

Reply:
Changed to footer-wrapper__link